Packs/SekoiaXDR/Integrations/SekoiaXDR/SekoiaXDR.py

```diff
@@ -0,0 +1,1526 @@
+
+            # ELSE
+            else:
+                readable_output = tableToMarkdown(
+                    f"{total} events out of "
+                    + str(events["total"])
+                    + f" retrieved for the {term}:",
+                    events["items"],
+                )
+
+            bool_flag = False
+        else:
+            demisto.debug(f"The query is still running with the status {status}")
+
+        time.sleep(int(args.get("interval_in_seconds") or 0))
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="SekoiaXDR.Events.Results",
+        outputs_key_field="search_job_uuid",
+        outputs=events["items"],
+    )
+
+
+def update_status_alert_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Parameters"""
+    alert_uuid, updated_status, comment = (
+        str(args.get("id")),
+        str(args.get("status")),
+        args.get("comment"),
+    )
+    sekoia_transition = STATUS_TRANSITIONS.get(updated_status)
+    readable_output = "Unknown alert"
+
+    workflow = client.get_workflow_alert(alert_uuid=alert_uuid)
+
+    for action in workflow["actions"]:
+        if action["name"] == sekoia_transition:
+            update = client.update_status_alert(
+                alert_uuid=alert_uuid, action_uuid=action["id"], comment=comment
+            )
+            if update or update == {}:
+                readable_output = (
+                    "### Alert " + alert_uuid + " updated to status: " + updated_status
+                )
+            else:
+                raise DemistoException(
+                    "Failure to update the status of the alert. \
+                    Run the command !sekoia-get-workflow-alert to see the possible\
+                    transitions and review the code."
+                )
+
+    return CommandResults(readable_output=readable_output, outputs=update)
+
+
+def post_comment_alert_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Parameters"""
+    alert_uuid, comment, author = (
+        str(args.get("id")),
+        str(args.get("comment")),
+        str(args.get("author")),
+    )
+
+    response = client.post_comment_alert(
+        alert_uuid=alert_uuid, content=comment, author=author
+    )
+    readable_output = tableToMarkdown(
+        f"Alert {alert_uuid} updated with the comment: \n {comment}:", response
+    )
+
+    return CommandResults(readable_output=readable_output, outputs=response)
+
+
+def get_comments_alert_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Parameters"""
+    alert_uuid = str(args.get("id"))
+
+    response = client.get_comments_alert(alert_uuid=alert_uuid)
+
+    for item in response["items"]:
+        # Add author of the comment
+        if item["author"].startswith("user"):
+            user = client.get_user(user_uuid=item["created_by"])
+            item["user"] = f"{user.get('firstname')} {user.get('lastname')}"
```

Comment:
Inside item["author"] there's who commented the alert, and there's several possibilities. From them there's a value with `user:<user_id>`.  And I think that it will be good to show name not the id. but I'm also agree with you there will be a lot of api calls. so I have delete this line.